scripts/webidl/specs/events/EventTarget.idl

```diff
@@ -0,0 +1,6 @@
+[Exposed=(Window,Worker)]
```

Comment:
Would prefer this to live in living or living/dom or something. Again probably nicer next to EventTarget-impl.js.